static-responses/README.md

```diff
@@ -104,7 +106,8 @@ $ curl localhost:8080/app/conf
 {
   "debug": true,
   "env": "local",
-  "version": "1.0"
+  "version": "1.0",
+  "couper": "1.3.1"
```

Comment:
`couper` will probably be the first key.

Reply:
:heavy_check_mark: